.eslintrc.json

```diff
@@ -24,7 +24,7 @@
         "import/no-named-as-default": "off",
         "import/no-unresolved": "warn",
         "no-mixed-spaces-and-tabs": 0,
-        "no-console": 2,
+        "no-console": 0,
```

Comment:
The number is for how many the linter will accept. If you want to turn it off, change `0` to `"off"`

Reply:
oh, ok